src/components/Menu/index.jsx

```diff
@@ -0,0 +1,37 @@
+import './Menu.scss';
+import React from 'react';
+import { FaHome } from 'react-icons/fa';
+import { FaFlask } from 'react-icons/fa';
+import { FaAngleDoubleRight } from 'react-icons/fa';
+import { FaRocket } from 'react-icons/fa';
+import { FaArrowDown } from 'react-icons/fa';
+
+class Menu extends React.Component {
+  render() {
+    return (
+      <div className="wrapper-menu">
+        <div className="home-icon">
+          <FaHome />
+        </div>
+        <div className="projects">
```

Comment:
```suggestion
        </div>
 
        <div className="projects">
```
Các block khác nhau viết tách nhau ra cho dễ nhìn, nhắc bao nhiêu lần rồi, không được viết tất cả dính hết vào nhau